### PR TITLE
Chore/design tokens

### PR DESCRIPTION
**Pull request by wanbinkimoon**
chore: implemented new design tokens 

----

### Motivation and Context

maintenance

### Priority

- [ ] 1 - Highest
- [x] 2 - High
- [ ] 3 - Medium
- [ ] 4 - Low
- [ ] 5 - Lowest
- [ ] 6 - Not a Priority

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Component (add a Component as approved dy Design System)
- [ ] Docs (add documentation)
- [x] Chore (changes that adds small enhancement)
- [x] Breaking (fix or feature that would cause existing functionality to not work as expected)

### Abstract

N/A

### Screenshots
N/A

### Note

----

### CODE REVIEW

File: src/components/panel/z-panel-elem/styles.css [hunk 1]
```diff
@@ -23,8 +21,8 @@
 }
 
 .panel-elem-link {
-  border-radius: var(--radius-min);
-  margin: var(--basex1) auto 0;
+  border-radius: var(----border-no-radius);
```
Comment (ufollettu) [inline src/components/panel/z-panel-elem/styles.css:24]: fix extra `--`

File: src/components/card/z-card-header/styles.css [hunk 2]
```diff
@@ -3,46 +3,46 @@ header {
...
   width: auto;
-  height: var(--basex6);
+  height: calc(var(--space-unit) * 6);
   position: relative;
-  border-bottom: var(--border-base) solid var(--bg-grey-200);
-  border-radius: var(--radius-base) var(--radius-base) var(--radius-min)
-    var(--radius-min);
+  border-bottom: var(--border-size-medium) solid var(--bg-grey-200);
+  border-radius: var(--border-radius) var(--border-radius) var(----border-no-radius)
+    var(----border-no-radius);
```
Comment (ufollettu) [inline src/components/card/z-card-header/styles.css:10]: fix extra `--`

File: src/components/card/z-card-header/styles.css [hunk 3]
```diff
@@ -3,46 +3,46 @@ header {
...
   align-items: flex-start;
   width: auto;
-  height: var(--basex6);
+  height: calc(var(--space-unit) * 6);
   position: relative;
-  border-bottom: var(--border-base) solid var(--bg-grey-200);
-  border-radius: var(--radius-base) var(--radius-base) var(--radius-min)
-    var(--radius-min);
+  border-bottom: var(--border-size-medium) solid var(--bg-grey-200);
+  border-radius: var(--border-radius) var(--border-radius) var(----border-no-radius)
```
Comment (ufollettu) [inline src/components/card/z-card-header/styles.css:9]: fix extra `--`

File: src/index.ts [hunk 4]
```diff
@@ -1 +1,2 @@
+import "@zanichelli/albe-design-tokens/dist/tokens.css";
```
Comment (DarioSacco) [inline src/index.ts:1]: Why this import? Tokens are already imported in `global.css`

File: src/global.css [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (DarioSacco) [inline src/global.css:1]: there is a benefit on importing the tokens in `index.ts`? I think the right import was here

File: stencil.config.ts [hunk 6]
```diff
@@ -24,5 +24,5 @@ export const config: Config = {
       serviceWorker: null, // disable service workers
     },
   ],
-  globalStyle: "src/globals/global-styles.css"
+  globalStyle: "src/global.css"
```
Comment (DarioSacco) [inline stencil.config.ts:27]: this becomes useless if the `global.css` file is left empty

**Review by DarioSacco — CHANGES_REQUESTED** [review]

**Comment by ufollettu** [comment]
### On design-tokens library:

- grid.scss and text.scss should not be legacy but present tokens
- the dist folder should not be included in the sources but compiled every time `build` command is run, and only the dist should be the contents of the package published on NPM, furthermore the dist folder should be emptied before each build
- the build and prepublish commands included in package.json include yarn as a dependency, but yarn is not specified in the devDependencies or library requirements in the Readme. It would be correct to implement generic commands and leave the choice of the task runner to whoever uses the package, or to make explicit its dependency
```  "scripts": {
    "release": "standard-version",
    "build:themes:default": "sass ./src/modules/themes/default.scss ./dist/themes/default.css",
    "build:themes": "yarn build:themes:default",
    "build:tokens": "sass ./src/main.scss ./dist/tokens.css",
    "build": "yarn build:tokens && yarn build:themes",
    "prepublish": "yarn build"
  },
  "devDependencies": {
    "sass": "^1.32.8",
    "standard-version": "^9.1.1"
  }
```
- yarn-error.log file should be ignored by .git

File: src/global.css (hunk 5 above)
Comment (wanbinkimoon) [reply src/global.css:1]: nope I think we ( I ) removed the wrong line ...

File: stencil.config.ts (hunk 6 above)
Comment (wanbinkimoon) [reply stencil.config.ts:27]: thi is useful considering the commenti right above

**Comment by wanbinkimoon** [comment]
> ### On design-tokens library:
> * grid.scss and text.scss should not be legacy but present tokens
> * the dist folder should not be included in the sources but compiled every time `build` command is run, and only the dist should be the contents of the package published on NPM, furthermore the dist folder should be emptied before each build
> * the build and prepublish commands included in package.json include yarn as a dependency, but yarn is not specified in the devDependencies or library requirements in the Readme. It would be correct to implement generic commands and leave the choice of the task runner to whoever uses the package, or to make explicit its dependency
> 
> ```
>     "release": "standard-version",
>     "build:themes:default": "sass ./src/modules/themes/default.scss ./dist/themes/default.css",
>     "build:themes": "yarn build:themes:default",
>     "build:tokens": "sass ./src/main.scss ./dist/tokens.css",
>     "build": "yarn build:tokens && yarn build:themes",
>     "prepublish": "yarn build"
>   },
>   "devDependencies": {
>     "sass": "^1.32.8",
>     "standard-version": "^9.1.1"
>   }
> ```
> 
> * yarn-error.log file should be ignored by .git

1. why do you think we should treat them as present tokens? they are part of the legacy implementation, i'd rather use them from the legacy folder also as a a reminder that this piece of DS must be tested again both in design and development 
2. correct me if I'm wrong, I think that the dist folder is needed because is generated in build and `npm publish` do not build the package. Not sure about this and cannot find proper documentation on this 
3. ok 
4. ok

**Comment by wanbinkimoon** [comment]
@ufollettu found proper docs about point 2 we can add `/dist` in `.gitignore`

**Comment by ufollettu** [comment]
> > ### On design-tokens library:
> > 
> > * grid.scss and text.scss should not be legacy but present tokens
> > * the dist folder should not be included in the sources but compiled every time `build` command is run, and only the dist should be the contents of the package published on NPM, furthermore the dist folder should be emptied before each build
> > * the build and prepublish commands included in package.json include yarn as a dependency, but yarn is not specified in the devDependencies or library requirements in the Readme. It would be correct to implement generic commands and leave the choice of the task runner to whoever uses the package, or to make explicit its dependency
> > 
> > ```
> >     "release": "standard-version",
> >     "build:themes:default": "sass ./src/modules/themes/default.scss ./dist/themes/default.css",
> >     "build:themes": "yarn build:themes:default",
> >     "build:tokens": "sass ./src/main.scss ./dist/tokens.css",
> >     "build": "yarn build:tokens && yarn build:themes",
> >     "prepublish": "yarn build"
> >   },
> >   "devDependencies": {
> >     "sass": "^1.32.8",
> >     "standard-version": "^9.1.1"
> >   }
> > ```
> > 
> > 
> > 
> > * yarn-error.log file should be ignored by .git
> 
> 1. why do you think we should treat them as present tokens? they are part of the legacy implementation, i'd rather use them from the legacy folder also as a a reminder that this piece of DS must be tested again both in design and development
> 2. correct me if I'm wrong, I think that the dist folder is needed because is generated in build and `npm publish` do not build the package. Not sure about this and cannot find proper documentation on this
> 3. ok
> 4. ok

1. I would have used another, more incremental approach: to consider current tokens that have not yet been revised and to move only those that have been revised to legacy. In practice, in the legacy folder I imagine to find the backward compatibility tokens, in the others those currently in use. The grid and text tokens are not subject to revision in this task, therefore I would consider them current until further implemented.
Or I'm missing something about the type of implementation you want to follow


1.

**Comment by wanbinkimoon** [comment]
@ufollettu seems a good approach i've changed the repo as suggested 
https://github.com/ZanichelliEditore/design-tokens/commit/afbe2350d8172d340b29f776792f20cef7e9c5a4

**Review by DarioSacco — APPROVED** [review]

**Review by ufollettu — APPROVED** [review]

**Pull request merged**